doc-deploy-dev/action.yml

```diff
@@ -18,12 +18,10 @@ inputs:
   workflow:
     description: 'Workflow name from where the artifacts are downloaded. By default, the same running workflow.'
     required: false
-    default: ""
     type: string
   repo:
     description: 'Repository name with owner (like actions/checkout)'
     required: false
-    default: ${{ github.repository }}
     type: string
   pr:
```

Comment:
I prefer not to, because of flexibility, and because they are already there.

@MaxJPRey and @jorgepiloto any opinions about this?

Reply:
The reason why we added additional parameters was because it was required in PyMAPDL. If, as of today, PyMAPDL only uses the "workflow" input parameter, I think it is fine if we remove non-used parameters.

If, at some point, any project requires the usage of other parameter, it will force us to explore why such need and we can always implement it back.

Reply:
I agree with @jorgepiloto - let's wait for @MaxJPRey's opinion.

Reply:
Let it be then! :)